src/Mod/CAM/Path/Tool/assets/manager.py

```diff
@@ -218,7 +217,6 @@ def _calculate_cache_key_from_construction_data(
     def _build_asset_tree_from_data_sync(
         self,
         construction_data: Optional[_AssetConstructionData],
-        store_name_for_cache: str,
     ) -> Asset | None:
```

Comment:
Lint is showing `unsupported operand type(s) for | (unsupported-binary-operation)`  
Should that be addressed in a separate PR ?